app/context/pizza/repo.ml

```diff
@@ -185,35 +185,29 @@ let find_pizza name =
 let find_pizzas_request =
```

Comment:
@timohuber This request returns either 1) one result or 2) none. The previous version with `LEFT JOIN` returns multiple rows, one row for each ingredient found. This has to be folded/collected into one pizza model with multiple ingredients.

Reply:
After a quick chat with Aron I agree that simplicity > performance concerns in this case :)